examples/next/src/components/BreadcrumbDemo.tsx

```diff
@@ -0,0 +1,20 @@
+import { usePathname } from "next/navigation";
+import { Breadcrumb, Icon } from "@animareflection/ui/client";
+import { FiChevronRight as ChevronRightIcon } from "react-icons/fi";
+
+import { Wrapper } from "components";
+
+const BreadcrumbDemo = () => {
+  const pathname = usePathname();
+  return (
+    <Wrapper title="Breadcrumb">
+      <Breadcrumb
+        pathname={pathname}
+        baseUrl="storybook.animareflection.com"
```

Comment:
A great spot for this hard-coded URL would be in `src/lib/app.ts`, I suggest (with the HTTPS prefix:

```ts
const app = {
  url: "https://storybook.animareflection.com",
  // ...
};
```